interactions.go

```diff
@@ -7,8 +7,230 @@ import (
 	"io"
 	"io/ioutil"
 	"net/http"
+	"time"
+	// "fmt"
 )
 
+// InteractionDeadline is a deadline for responding to an interaction, if you haven't responded in the time, you won't be able to respond later.
+const InteractionDeadline = time.Second * 3
+
+// ApplicationCommand is representing application's slash command.
+type ApplicationCommand struct {
+	ID            string                      `json:"id"`
+	ApplicationID string                      `json:"application_id,omitempty"`
+	Name          string                      `json:"name"`
+	Description   string                      `json:"description,omitempty"`
+	Options       []*ApplicationCommandOption `json:"options"`
+}
+
+// ApplicationCommandOptionType is type of an slash-command's option.
+type ApplicationCommandOptionType uint8
+
+// Application command option types.
+const (
+	ApplicationCommandOptionSubCommand = ApplicationCommandOptionType(iota + 1)
+	ApplicationCommandOptionSubCommandGroup
+	ApplicationCommandOptionString
+	ApplicationCommandOptionInteger
+	ApplicationCommandOptionBoolean
+	ApplicationCommandOptionUser
+	ApplicationCommandOptionChannel
+	ApplicationCommandOptionRole
+)
+
+// ApplicationCommandOption is representing an option/subcommand/subcommands group.
+type ApplicationCommandOption struct {
+	Type        ApplicationCommandOptionType `json:"type"`
+	Name        string                       `json:"name"`
+	Description string                       `json:"description,omitempty"`
+	// Default     bool                              `json:"default"`
```

Comment:
remove

Reply:
No, I won't, I specially marked it as comment, because it hasn't present yet in the docs (I discovered it earlier, but developers decided to delete this from docs, so I kept this here but under comment, until this will appear on the official docs).